iothub/service/design.md

```diff
@@ -0,0 +1,119 @@
+# RBAC API Design for IoT hub
+
+## Service Client
+
+```csharp
+/// <summary>
+/// Create ServiceClient using AAD token credentials and the specified Transport Type.
```

Comment:
you can use lower case when referring to things generically, like transport type.